Makefile

```diff
@@ -4,12 +4,20 @@ all: generate format
 
 generate:
 	@mkdir -p ./pb
+	@./merge-proto.sh \
+		fig/tendermint/codec/v1/codec.proto \
+		fig/tendermint/transform/v1/transform.proto \
+		fig/tendermint/liquidity/v1/liquidity.proto
```

Comment:
Isn't this needed only for message decoding (which happens in the subgraph)?

There should be a clear separation between protobufs for Cosmos Firehose and the ones for message decoding (which are not used anywhere in the Firehose stack), as they have completely different workflows. There should be at least a separate directory where we put them.